libs/runtime_configuration/src/runtime_configuration.cpp

```diff
@@ -73,6 +73,21 @@ namespace hpx { namespace parcelset {
 }}    // namespace hpx::parcelset
 
 namespace hpx { namespace util {
+
+    namespace detail {
+
+        // CMake does deal with explicit semicolons well, for this reason, the
```

Comment:
I most likely meant 'does _not_ deal well' *grumble*

Reply:
I think you clearly wrote that comment sarcastically ;)